### DISCUSSION TITLE
Coverage map on a non-flat terrain

### DISCUSSION THREAD

**Discussion by Stan121984** [NVlabs/sionna#213] [Q&A]
Hi, I am working on a scene that has been created using Blender+Blosm. I import both buildings and the terrain. Therefore the ground is not flat.
Is there any way to compute a coverage map that is not defined as a flat surface, but as a 3d object (e.g always 1.5m above the ground)?
If not, maybe you plan to enable such a feature in further releases?
Thanks in advance!

**Comment by jhoydis** [comment]
Hi,

It is currently not possible to compute coverage maps on non-planar surfaces. Although it would be in principle possible to compute such maps on any possible mesh, we currently do to plan to implement such a feature.

**Reply by Stan121984** [reply]
Thank you for the clarification!
